db/seeds/activities/mylearning.rb

```diff
@@ -182,6 +224,66 @@
 
 ########################################################################################################################
 
+a = Activity.find_or_create_by(stem_course_template_no: '8dbe73ec-cd3d-ee11-bdf4-002248c6f9ce') do |activity|
+  activity.title = 'Understanding Maths and Logic in Computer Science'
+  activity.credit = 20
+  activity.slug = 'understanding-maths-and-logic-in-computer-science'
+  activity.category = 'online'
+  activity.self_certifiable = false
+  activity.provider = 'stem-learning'
+  activity.stem_activity_code = 'CO213'
+  activity.always_on = true
+end
+
+a.programmes << secondary_certificate unless a.programmes.include?(secondary_certificate)
+
+########################################################################################################################
+
+a = Activity.find_or_create_by(stem_course_template_no: '716d350d-ce3d-ee11-bdf4-002248c6f9ce') do |activity|
+  activity.title = 'An Introduction to Computer Networking for Teachers'
+  activity.credit = 20
+  activity.slug = 'an-introduction-to-computer-networking-for-teachers'
+  activity.category = 'online'
+  activity.self_certifiable = false
+  activity.provider = 'stem-learning'
+  activity.stem_activity_code = 'CO214'
+  activity.always_on = true
+end
+
+a.programmes << secondary_certificate unless a.programmes.include?(secondary_certificate)
```

Comment:
Hi, looks like you have put CO213 and CO214 in the secondary certificate, but[ the ticket ](https://github.com/NCCE/teachcomputing.org-issues/issues/2455)asks for them to be in the CSA. Please check.